packages/vaex-ml/vaex/ml/catboost.py

```diff
@@ -97,17 +96,14 @@ def fit(self, df, evals=None, early_stopping_rounds=None, verbose_eval=None, plo
         :param bool plot: if True, display an interactive widget in the Jupyter
             notebook of how the train and validation sets score on each boosting iteration.
         '''
-        # Ensure strings
-        target = _ensure_strings_from_expressions(self.target)
-        features = _ensure_strings_from_expressions(self.features)
 
-        data = df[features].values
-        target_data = df[target].values
+        data = df[self.features].values
```

Comment:
If/when we move to arrow, `.values` might return an arrow array. I think they automatically cast to a numpy array, and I think it's fine that we attach the problem when it appears, just a heads up.